crates/delta/src/routes/servers/server_edit.rs

```diff
@@ -148,6 +156,21 @@ pub async fn edit(
         server.banner = partial.banner.clone();
     }
 
+    // 5. Transfer ownership
+    if let Some(owner) = owner {
+        let owner_reference = Reference { id: owner.clone() };
+        // Check if member exists
+        owner_reference.as_member(db, &server.id).await?;
+        let owner_user = owner_reference.as_user(db).await?;
+
+        if owner_user.bot.is_some() {
+            return Err(create_error!(InvalidOperation));
```

Comment:
If the owner is a bot, shouldn't this throw the `IsBot` error? Or is `InvalidOperation` more suitable for this? 🤷🏻

Reply:
Afaik `isBot` is for user-only endpoints that cannot be used by bots, but im not 100% sure about it

Reply:
Ohhhh I see. If you check the file where that error is defined (or whatever it is in Rust), there are comments defining different sections of errors. Can't remember the file or category comment for `IsBot`, so you'll have to check yourself lmao

Reply:
It does not has comment explaining it's purpose [where it was defined](https://github.com/revoltchat/backend/blob/bcf6561c0c03c45081dd24b3ec100714420d0d8e/crates/core/result/src/lib.rs#L123), but according to its [usage](https://github.com/revoltchat/backend/blob/bcf6561c0c03c45081dd24b3ec100714420d0d8e/crates/delta/src/routes/channels/message_search.rs#L23), i think its purpose to prevent bots from using user-only endpoints